fedora/client/bodhi.py

```diff
@@ -17,6 +17,10 @@
 # License along with python-fedora; if not, see <http://www.gnu.org/licenses/>
 #
 """
+DEPRECATED: This module is deprecated and has moved to bodhi.client.bindings.
+As the bodhi-0.9 client still uses it, it will remain here until Fedora 25 is
+EOL.
+
```

Comment:
@pypingou I don't intend to do that - it is backwards incompatible and I don't want to assume that developers haven't scripted against the client. I plan to just let Fedora 25 go EOL with bodhi 0.9, and then consider 0.9 fully retired.